bitrise.yml

```diff
@@ -0,0 +1,52 @@
+format_version: "10"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ""
+workflows:
+  audit-this-step:
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            stepman audit --step-yml ./step.yml
+  test:
+    before_run:
+    - audit-this-step
+    steps:
+    - go-test:
+        inputs:
+        - packages: |-
+            github.com/bitrise-steplib/step-custom-test-results-export/testresultexport
```

Comment:
we have a legacy: `steps-` and the new `bitrise-step-` prefix for our step repos. please keep on going with the new one.
please dont forget to rewerite everywhaere